content/chat/moderation/index.textile

```diff
@@ -15,6 +15,10 @@ Moderation strategies can take many forms. Human moderators can sit and particip
 
 Ably Chat supports a variety of moderation options in chat rooms, to help you keep your participants safe and engaged.
 
+h2(#getting-started). Getting started
+
+All moderation rules can be set up in the integration rules section of your Ably dashboard and in the control API.
+
```

Comment:
I'm not sure it's worth including this section in the overview. 

Instead I think this should be more explicit under the 'rules fields' that you have on the relevant pages.

We can also include a simple cUrl command for using the Control API to do the same thing.

Reply:
4e2ed73fa